source/transactions/transactions.rst

```diff
@@ -580,6 +580,13 @@ transaction, the server will return an error. The ``writeConcern`` argument
 of the commitTransaction and abortTransaction commands has
 semantics analogous to existing write commands.
 
+Behavior of the recoveryToken field
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Only included for sharded transactions and only when running a
+commitTransaction or abortTransaction command. See the
```

Comment:
recoveryToken is only for commitTransaction. For that reason, drivers should only expect recoveryToken to exists in mongos response if ok: 1

Reply:
The section below says:
```
When a driver runs a command within a transaction, the mongos response
includes a ``recoveryToken`` field.
```
Do you want me to change it to say only (ok:1) responses include it? In my testing command errors also include a recoveryToken field.

Reply:
It is not guaranteed to be there if the response is not ok: 1. In general, these are the cases when mongos caused the error (as opposed to errors from mongod bubbled up to mongos).

Reply:
I updated to say: `Every successful (``ok:1``) command response in a sharded transaction includes a ``recoveryToken`` field. ...`

Should drivers also look for a `recoveryToken` within `ok:0` responses? I could add:
`
Every successful (``ok:1``) command response in a sharded transaction includes a ``recoveryToken`` field. Any unsuccessful (``ok:0``) command responses in a sharded transaction MAY include a ``recoveryToken`` field. ...`

Reply:
For now, it is not needed for drivers to look for recoveryToken since it cannot commit the transaction anyway. The suggested text looks good to me.

Reply:
Perfect, thanks.